src/adv.py

```diff
@@ -33,19 +65,269 @@
+    # elif action.lower() in ['inspect', 'interact', 'look', 'examine']:
+    #     self.player_examine(action.lower())
+
+
+def take_item(item):
+    #     for x in arr:
+    #   if item in x.values():
+    # print(x.values())
+    item_present = False
+    for i in my_player.current_room.items:
+        if i.name == item:
+            my_player.inventory.append(items[item])
+            items[item].on_take()
+            my_player.current_room.items.remove(items[item])
+            item_present = True
+        # else:
+    if not item_present:
+        print('There is nothing by that name in this location')
+
+    # my_player.inventory.append(items[item])
+    # my_player.current_room.items.remove(items[item])
+
+
+def drop_item(item):
+    my_player.current_room.items.append(items[item])
+    items[item].on_drop()
+    my_player.inventory.remove(items[item])
+
+
+def no_access():
+    print('\n' + 'There is nothing in that direction to move to')
+    prompt(my_player)
+
+
+def player_move(direction):
+    # ask = 'Where would you like to move to?\n'
+    if direction in ['up', 'north', 'n']:
+        if hasattr(my_player.current_room, 'n_to'):
+            direction = my_player.current_room.n_to
+            movement_handler(direction)
+        else:
+            no_access()
+    elif direction in ['left', 'west', 'w']:
+        if hasattr(my_player.current_room, 'w_to'):
+            direction = my_player.current_room.w_to
+            movement_handler(direction)
+        else:
+            no_access()
+    elif direction in ['right', 'east', 'e']:
+        if hasattr(my_player.current_room, 'e_to'):
+            direction = my_player.current_room.e_to
+            movement_handler(direction)
+        else:
+            no_access()
+    elif direction in ['down', 'south', 's']:
+        if hasattr(my_player.current_room, 's_to'):
+            direction = my_player.current_room.s_to
+            movement_handler(direction)
+        else:
+            no_access()
+
+
+def movement_handler(destination):
+    print(f'\n You have moved to the {destination.name}')
+    my_player.current_room = destination
+    os.system('clear')
+    header()
+    print_location()
+
+
+def main_game_loop(player_name):
+    my_player.name = player_name
+    while my_player.is_playing:
+        prompt(my_player)
+    # * Prints the current room name
+    # * Prints the current description (the textwrap module might be useful here).
+    # * Waits for user input and decides what to do.
+    #
+    # If the user enters a cardinal direction, attempt to move to the room there.
+    # Print an error message if the movement isn't allowed.
+    #
+    # If the user enters "q", quit the game.
+
+    if __name__ == '__main__':
+        main_game_loop(player_name)
+
+
+#######################################
+#  Terminal Output Helpers
+#######################################
+
+
+def title_screen_selections():
```

Comment:
This is the second place you're asking for input. The overall logic of the game would be easier to follow if it was in one place.